codes/classical/bits/gold.yml

```diff
@@ -0,0 +1,51 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# Use UTF-8 unicode encoding
+# AMS-TeX commands are rendered inside \( ... \) using MathJaX.
+# Allowed external bibliographic references are
+#    \cite{arXiv:#.#} or \cite{arXiv:quant-ph/#} (PREFERRED),
+#    \cite{doi:#}, or, as a last resort
+#    \cite{manual:{(enter citation line incl. author and year here)}}
+# External websites such as code tables, coding theory packages, github pages linked as
+#    \url{https://example.com/example}
+#    \href{https://example.com/example}{link text}
+# Internal references to codes are
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
+code_id: gold
+physical: bits
+logical: bits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title (SINGULAR) + first reference(s) (optional).
+name: 'Gold'
+introduced: '\cite{doi:10.1109/TIT.1967.1054048}'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: 'A \([2^r -1, 2r ]\) is a cyclic linear code characterized by the generator polynomial of degree r of a preferred-pair of m-sequences.'
```

Comment:
Replace "preferred-pair of m-sequences." with "two maximum-period sequences of period 2^r-1 with absolute cross-correlation ≤ 2^((r+2)/2)."